libraries/protocol/include/scorum/protocol/asset.hpp

```diff
@@ -86,6 +93,11 @@ struct asset
         FC_ASSERT(a.symbol == b.symbol);
         return asset(a.amount + b.amount, a.symbol);
     }
+    friend asset operator*(const asset& a, const asset& b)
```

Comment:
add unit test for a new function.

Reply:
This operation is not correct. SCR*SCR=SCR^2, but operation returns SCR.